samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetectionML.ConsoleApp/ModelBuilder.cs

```diff
@@ -0,0 +1,73 @@
+// This file was auto-generated by ML.NET Model Builder. 
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using StopSignDetectionML.Model;
+namespace StopSignDetectionML.ConsoleApp
+{
+    public static class ModelBuilder
+    {
+        private static string TRAIN_DATA_FILEPATH = @"C:\Users\brachtma\AppData\Local\Temp\fe343e6e-b4bb-4eae-94e3-ade20e72b9a0.csv";
```

Comment:
This file path isn't going to work correctly.

Reply:
I meant to delete the console app. I removed from Solution Explorer but didn't delete I guess.

Reply:
Fixed with https://github.com/dotnet/machinelearning-samples/pull/882/commits/5b830114cf7e91b2b9f294aa6f38240e582d6e57